scapy/contrib/gtp_v2.py

```diff
@@ -262,14 +265,18 @@ def answers(self, other):
                 self.payload.answers(other.payload))
 
 
-class IE_IPv4(gtp.IE_Base):
-    name = "IE IPv4"
+class IE_IP_Address(gtp.IE_Base):
+    name = "IE IP Address"
     fields_desc = [ByteEnumField("ietype", 74, IEType),
-                   FieldLenField("length", None, length_of="address",
-                                 adjust=lambda pkt, x: x + 4, fmt="H"),
+                   FieldLenField("length", None, length_of="length",
```

Comment:
Can you use `LenField` instead

Reply:
I used FieldLenField because that what we are using in rest all IEs & fields
Do you still want me to change this? BTW, what's the difference between the two?

Reply:
FieldLen is specifically made to have the payload length. have a look at the source code